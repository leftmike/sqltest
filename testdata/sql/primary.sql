--
-- Test PRIMARY KEYs
--

DROP TABLE IF EXISTS tbl1;

CREATE TABLE tbl1 (
    c1 int primary key,
    c2 int,
    c3 int
);

INSERT INTO tbl1 VALUES
    (0, 10, 0),
    (1, 20, 0),
    (2, 30, 0),
    (3, 40, 0);

SELECT * FROM tbl1;

UPDATE tbl1 SET c1 = c1 * 10;

SELECT * FROM tbl1;
