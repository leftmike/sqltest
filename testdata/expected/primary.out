--
-- Test PRIMARY KEYs
--
DROP TABLE IF EXISTS tbl1;
CREATE TABLE tbl1 (
    c1 int primary key,
    c2 int,
    c3 int
);
INSERT INTO tbl1 VALUES
    (0, 10, 0),
    (1, 20, 0),
    (2, 30, 0),
    (3, 40, 0);
SELECT * FROM tbl1;
   c1 c2 c3
   -- -- --
 1  0 10  0
 2  1 20  0
 3  2 30  0
 4  3 40  0
(4 rows)
UPDATE tbl1 SET c1 = c1 * 10;
SELECT * FROM tbl1;
   c1 c2 c3
   -- -- --
 1  0 10  0
 2 10 20  0
 3 20 30  0
 4 30 40  0
(4 rows)
